lightning/src/ln/channelmanager.rs

```diff
@@ -3953,6 +3963,12 @@ where
 								node_id: channel.get_counterparty_node_id(),
 								msg: announcement_sigs,
 							});
+						} else if channel.is_usable() {
+							log_trace!(self.logger, "Sending funding_locked WITHOUT announcement_signatures but with channel_update for {}", log_bytes!(channel.channel_id()));
```

Comment:
Maybe add for "with channel_update for _private_", underscoring the privacy of the chan